gocourse02/main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+func main() {
+	zookeeper := Zookeeper{ID: 1, Name: "Jon Snow"}
+	cageTiger := &Cage{ID: 1, State: "close"}
+	cageCoyote := &Cage{ID: 2, State: "close"}
+	cageBeaver := &Cage{ID: 2, State: "close"}
```

Comment:
тут бажано використовувати константи, які задефайнені
```suggestion
	cageTiger := &Cage{ID: 1, State: stateClose}
	cageCoyote := &Cage{ID: 2, State: stateClose}
	cageBeaver := &Cage{ID: 2, State: stateClose}
```